app/assets/stylesheets/adl-overrides/adl-overrides.scss

```diff
@@ -535,3 +535,8 @@ body.works-show.adl_show {
     }
   }
 }
+
+// for pdf.js viewer
+.viewer-wrapper {
+  height: 900px;
+}
```

Comment:
Moved this override from viewer.scss so the rest of the file could be removed & just default to hyku's file.